src/pages/nfts/[id].vue

```diff
@@ -47,16 +47,18 @@ const hasTemplates = computed(() => {
   return nft.value?.extensions.includes('mintable_templates_limit')
 })
 
-const { error } = await useAsyncData(() => fetchNftDetails(route.params.id))
-
-if (error.value) {
-  throw showError({
-    data: {
-      entityId: route.params.id,
-      entityName: 'NFT',
-    },
-    statusMessage: 'EntityDetailsNotFound',
-  })
+try {
```

Comment:
Unrelated fix. I found that direct URL access to NFT detail page will cause infinite loading.
EG: https://aescan.dev.aepps.com/nfts/ct_2iJ5RHF2MbriceSetNiKARVii76Fr8Fi5wGY6kUHUj3wq7f3io
Could you help me to verify the solution @lukeromanowicz ?

Reply:
Thanks, I created the followup issue. Can you provide some hint how to improve it there? https://github.com/aeternity/aescan/issues/517 I am bit out of ideas. The 'correct' approach does not work when accessed directly. This solution it's not that effective but at least works